packages/blockchain-wallet-v4-frontend/webpack.config.js

```diff
@@ -195,8 +197,8 @@ module.exports = {
       'Content-Security-Policy': [
         "img-src 'self' data: blob:",
         "style-src 'self' 'unsafe-inline'",
-        'frame-src https://verify.isignthis.com/ https://wallet-helper.blockchain.info http://localhost:8081',
-        'child-src https://verify.isignthis.com/ https://wallet-helper.blockchain.info http://localhost:8081',
+        `frame-src ${iSignThisDomain} http://localhost:8081 ${envConfig.WALLET_HELPER_DOMAIN}`,
+        `child-src ${iSignThisDomain} http://localhost:8081 ${envConfig.WALLET_HELPER_DOMAIN}`,
```

Comment:
@prwelber  Can we get rid of localhost:8081 here too and just set it in wallet-options when we want it?

Reply:
Actually it comes from development config file